framework/include/cppmicroservices/Any.h

```diff
@@ -418,6 +418,19 @@ class US_Framework_EXPORT Any
     return _content ? _content->Type() : typeid(void);
   }
 
+  /**
+   * Parse a JSON string and return the results in an Any object. Hierarchical data is stored in
+   * maps, and lists are stored in vectors.
+   *
+   * @throws std::runtime_error with invalid JSON input
+   * @param  json a string containing JSON Data
+   * @param  use_ci_map_keys a bool which indicates whether or not the keys in any object maps should
+   *         be case insensitive. We use case insensitive keys in CppMicroServices, but there are
+   *         other uses of this static method that may not need them to be case insensitive.
+   * @return an Any object containing the parsed data. 
+   */
+  static Any FromJSON(std::string const& json, bool use_ci_map_keys = true);
+  static Any FromJSON(std::istream& json, bool use_ci_map_keys = true);
```

Comment:
as discussed offline; remove the optional `use_ci_map_keys` and default to case insensitive unordered map. if users need case sensitive and/or ordered maps, that can be exposed later.